README.md

```diff
@@ -237,10 +253,51 @@ redcarpet:
 
 **Pro tip:** If you're like us and prefer your navigation sticky, you can add `sticky_sidenav: true` on [page](#layout-page), [project](#layout-project), and [team member](#layout-team-member) layouts to have the sidenav follow as you scroll.
 
+### Hero
+
+```yml
+hero: # optional
+  image: /path/to/image.jpg # optional
+  callout:
+    alt: Callout white text! # optional
+    text: The rest of the callout
+  button: # optional
+    text: The button text
+    href: /button-href/
+```
+
+### Tagline intro
+
+```yml
+# optional, but must be used in conjunction with 'intro', below
+tagline: A tagline for your page
+# also optional, but must be used with 'tagline', above
+intro: |
+  Some introductory text content.
+
+  This will be processed as **Markdown**.
+```
+
+### Graphics list
+```yml
+# an optional list of graphics to display before or after the content
```

Comment:
Not a blocker since this isn't a change, but not obvious to me what this means. Different from a hero? Might be useful to have page(s) with examples of each that you can link to.

Reply:
The Graphics list is the dark section here with the 4 circle graphics https://designsystem.digital.gov/page-templates/landing/